src/App.tsx

```diff
@@ -55,30 +57,32 @@ const App = ({ skipLoadingScreen }: AppProps) => {
         'space-mono': require('assets/fonts/SpaceMono-Regular.ttf'),
         'montserrat-bold': require('assets/fonts/Montserrat-Bold.ttf'),
         'montserrat-italic': require('assets/fonts/Montserrat-Italic.ttf'),
-        'montserrat-regular': require('assets/fonts/Montserrat-Regular.ttf'),
-      }),
-    ]);
-  };
+        'montserrat-regular': require('assets/fonts/Montserrat-Regular.ttf')
+      })
+    ])
+  }
+
+  const theme = DarkModeTheme.theme
 
   const disableFontScaling = (): void => {
-    // @ts-ignore
+    // @ts-expect-error Error Occured
```

Comment:
You have typo `Occured` => `Occurred` 😄 
Replace this with the real message -> you can copy the message shown without annotation
```
@ts-expect-error Prop doesn't exist...
```